sources/packages/backend/package.json

```diff
@@ -74,14 +75,15 @@
     "passport-jwt": "^4.0.1",
     "pg": "^8.5.1",
     "qs": "^6.9.6",
-    "redis": "^4.6.6",
+    "redis": "^4.6.11",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
     "rxjs": "^7.5.4",
     "soap": "^1.0.0",
     "ssh2-sftp-client": "^9.1.0",
     "swagger-ui-express": "^4.3.0",
     "typeorm": "^0.3.17",
+    "workers": "^0.0.1",
```

Comment:
Which package is this exactly?

Reply:
Should have been a generated one, I have removed it now.